downstream/modules/platform/con-controller-scm-inventory-jobs.adoc

```diff
@@ -0,0 +1,9 @@
+[id="controller-scm-inventory-jobs"]
+
+= SCM inventory jobs
+
+When an inventory sourced from an SCM is executed, the full results automatically display in the *Output* tab. 
+This shows the same information you see if you run it through the Ansible command line, and can be useful for debugging.
```

Comment:
If used, the Ansible CLI displays the same information. This can be useful for debugging.